src/docs/user/references/VariablesReference.adoc

```diff
@@ -1,140 +1,183 @@
 This section describes useful variables, their scope, and their usage.
 
-[cols="1,1,1,1", options="header"]
+[cols="1,1,1,1,1", options="header"]
```

Comment:
Could be really helpful to define before the table what you mean by _all scripts_.